CppcheckTargets.cmake

```diff
@@ -113,9 +123,7 @@ function(add_cppcheck_sources _targetname)
 			FAIL_REGULAR_EXPRESSION
 			"${CPPCHECK_FAIL_REGULAR_EXPRESSION}")
 
-		add_custom_command(TARGET
-			all_cppcheck
-			PRE_BUILD
+		add_custom_target(${_targetname}_cppcheck
```

Comment:
Thanks - this setup of having multiple targets seems like a much more reliable solution than pre-build commands on one target.